src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -0,0 +1,67 @@
+package ru.skypro.homework.controller;
+
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.homework.dto.AdDto;
+import ru.skypro.homework.dto.AdsDto;
+import ru.skypro.homework.dto.CreateAdDto;
+import ru.skypro.homework.dto.ExtendedAdDto;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/ads")
+@CrossOrigin(value = "http://localhost:3000")
+public class AdsController {
+
+    @GetMapping
+    public AdsDto getAllAds() {
+        return new AdsDto();
+    }
+
+    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public AdDto addAd(@RequestBody CreateAdDto ad,
```

Comment:
```suggestion
    public AdDto addAd(@RequestPart("properties") CreateAdDto ad,
```
аннотация неправильная, также как и название части мультипартового запроса